src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -2826,319 +2776,147 @@ void InitZoneAirSetPoints(EnergyPlusData &state)
-                                    OutputProcessor::SOVTimeStepType::System,
-                                    OutputProcessor::SOVStoreType::Summed,
-                                    Zone(Loop).Name);
-                SetupOutputVariable(state,
-                                    "Zone Air System Latent Heating Rate",
-                                    OutputProcessor::Unit::W,
-                                    state.dataHeatBal->latentReports(Loop).ZoneLTLoadHeatRate,
-                                    OutputProcessor::SOVTimeStepType::System,
-                                    OutputProcessor::SOVStoreType::Average,
-                                    Zone(Loop).Name);
-                SetupOutputVariable(state,
-                                    "Zone Air System Latent Cooling Rate",
-                                    OutputProcessor::Unit::W,
-                                    state.dataHeatBal->latentReports(Loop).ZoneLTLoadCoolRate,
-                                    OutputProcessor::SOVTimeStepType::System,
-                                    OutputProcessor::SOVStoreType::Average,
-                                    Zone(Loop).Name);
-                // temporarily hide these behind DoLatentSizing flag
-                SetupOutputVariable(state,
-                                    "Zone Air System Sensible Heat Ratio",
-                                    OutputProcessor::Unit::None,
-                                    state.dataHeatBal->latentReports(Loop).ZoneSensibleHeatRatio,
-                                    OutputProcessor::SOVTimeStepType::System,
-                                    OutputProcessor::SOVStoreType::Average,
-                                    Zone(Loop).Name);
-                SetupOutputVariable(state,
-                                    "Zone Air Vapor Pressure Difference",
-                                    OutputProcessor::Unit::Pa,
-                                    state.dataHeatBal->latentReports(Loop).ZoneVaporPressureDifference,
-                                    OutputProcessor::SOVTimeStepType::System,
-                                    OutputProcessor::SOVStoreType::Average,
-                                    Zone(Loop).Name);
+        for (int zoneNum = 1; zoneNum <= NumOfZones; ++zoneNum) {
+            auto &thisZone = state.dataHeatBal->Zone(zoneNum);
+            state.dataZoneTempPredictorCorrector->zoneHeatBalance(zoneNum).setUpOutputVars(state, DataStringGlobals::zonePrefix, thisZone.Name);
+            if (state.dataHeatBal->doSpaceHeatBalanceSizing || state.dataHeatBal->doSpaceHeatBalanceSimulation) {
+                for (int spaceNum : state.dataHeatBal->Zone(zoneNum).spaceIndexes) {
+                    state.dataZoneTempPredictorCorrector->spaceHeatBalance(spaceNum).setUpOutputVars(
+                        state, DataStringGlobals::spacePrefix, state.dataHeatBal->space(spaceNum).Name);
+                }
+            }
```

Comment:
Example output variable setup. Only setup the space outputs if either `doSpaceHeatBalanceSizing` or `doSpaceHeatBalanceSimulation` is requested.